ipaclient/plugins/vault.py

```diff
@@ -752,8 +751,7 @@ def forward(self, *args, **options):
                 error=_('Invalid vault type'))
 
         # initialize NSS database
-        current_dbdir = paths.IPA_NSSDB_DIR
```

Comment:
NACK, `current_dbdir` is a global variable, the value must be changed here otherwise NSS initialization might explode later.